config/monitor/config.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitor
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-jet-azure/apis/rconfig"
+	"github.com/crossplane-contrib/provider-jet-azure/config/common"
+)
+
+const groupLog = "monitor"
+
+// Configure configures monitor group
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("azurerm_monitor_metric_alert", func(r *config.Resource) {
+		r.Kind = "MetricAlert"
+		r.ShortGroup = groupLog
+		r.References = config.References{
```

Comment:
And configuring references to `action_group_id`s?

Reply:
As there is currently no resource id reference support in Terrajet, a reference is not configured for this field. However, I have opened an issue on this topic. Please see: https://github.com/crossplane-contrib/provider-jet-azure/issues/95
(I also added a TODO comment for this)